ui/src/app/login/controllers/LoginCtrl.js

```diff
@@ -9,13 +9,16 @@
 
     $scope.login = function(){
       $scope.incorrectCredentials = false;
+      $scope.unApproved = false;
       UserService.login($scope.user, function(status) {
-        if (status){
+        if (status.state == 200) {
```

Comment:
Yeah, but comes with a price too, it's an extra step for JavaScript to check the type so it affects performance.